<?xml version="1.0"?>
<launch>

    <node name="grasping_test" pkg="cob_grasping" type="grasping_test.py" output="screen">
        <param name="scene" value="table" />
        <param name="switch_arm" value="True" />
    </node>

</launch>
